### PR TITLE
Dockerfile: make edgefs build in containter be working

### PR DESCRIPTION
**Pull request by inevity**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.
- [ ] Change modifies documentation or comments only.

#### Description
Make the Dockerfile working.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

**Comment by dyusupov** [comment]
:star: PR built and available in a preview environment **nexenta-edgefs-pr-326**

File: Dockerfile [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
 FROM ubuntu:16.04 as builder
 
+ENV DEBIAN_FRONTEND noninteractive
+
 RUN apt-get update &&\
     apt-get install -y software-properties-common python-software-properties &&\
     add-apt-repository ppa:git-core/ppa &&\
     apt-get update &&\
-    apt-get install -y git make
+    apt-get install -y -qq --no-install-recommends git make gcc libssl-dev
```
Comment (dyusupov) [inline Dockerfile]: git and make should be the only 2 dependencies that we need. Installation of gcc libssl-dev should be handled already in top-level Makefile rules. Please check.

Comment (inevity) [reply Dockerfile]: Yes, Since have add 'make clean' , the make world will install the gcc and libssl-dev depency.

Comment (inevity) [reply Dockerfile]: Now, remove the depency and repull request

**Pull request merged**